format/Format.py

```diff
@@ -497,28 +497,34 @@ def _end(self):
 
     def _goniometer(self):
         """Overload this method to read the image file however you like so
-        long as the result is an goniometer."""
+        long as the result is a goniometer."""
         return None
 
     def _detector(self):
         """Overload this method to read the image file however you like so
-        long as the result is an detector."""
+        long as the result is a detector."""
         return None
 
     def _beam(self):
         """Overload this method to read the image file however you like so
-        long as the result is an beam."""
+        long as the result is a beam."""
         return None
 
     def _scan(self):
         """Overload this method to read the image file however you like so
-        long as the result is an scan."""
+        long as the result is a scan."""
         return None
 
     def get_static_mask(self):
         """Overload this method to override the static mask."""
         return None
 
+    def get_spectrum(self):
+        """ Overload this method to read the image file however you like so
```

Comment:
image file -> data file for clarity?

Reply:
`Image file` is used extensively in the code as is.